packages/components/src/components/radio-button/radio-button.stories.ts

```diff
@@ -31,142 +32,87 @@ export default {
  */
 
 export const Default = {
+  name: 'Default',
   render: (args: any) => {
     return generateTemplate({ args });
   }
 };
 
 /**
- * Use the `size` attribute to change the size of the input radio. This attribute affects the font-size within the element, while the element itself remains the same size.
+ * Use the `size` attribute to change the size of the input radio.
```

Comment:
Use the `size` attribute to change the size.

Reply:
Updated